Misc/NEWS.d/next/Documentation/2019-05-08-20-06-22.bpo-36620.5d3gYv.rst

```diff
@@ -0,0 +1 @@
+Update itertools documentation with optional args registration.
```

Comment:
Can you actually remove the NEWS entry? For small documentation fixes an NEWS entry isn't really needed, and can end up cluttering up the news at the end of the day.

Reply:
@lisroach should be removed! I thought it was required because the CI checks had a large red X on not having a NEWS entry filled out, then the category disappeared after I filled it out (not sure if it's because I filled it out and it completed or if the CI checks deleted the category after PR instantiation).

Reply:
Okay I get it now; the .rst file must be present but the entry is "skip news" for the bot to skip news. Updated.